modules/onMath/index.js

```diff
@@ -24,13 +26,16 @@ for(var i=0;i<mathKeysToGet.length;i++) {
   mathItems[mathKeysToGet[i]] = mathjs[mathKeysToGet[i]];
 }
 
+//
+// INPUT FULTERS
+// 
 var mathKeys = Object.keys(mathItems);
 var mathSymbols = ".,*+-/()%=";
 
 var onlySymbols = new RC("^[\\s" + REEscape(mathSymbols) + "]*$");
 var onlyNumbers = new RC(/^[\.\s\d]*$/);
 var onlyKeys = new RC("^\\s*((" + mathKeys.join(")|(") + "))+\\s*$");
-var onlyTime = new RC(/^\s*([0-9]+:?)+\s*$/);
+var onlyTime = new RC(/^\s*(:?[0-9]+:?)+\s*$/);
```

Comment:
Rather than `\s*`, I'd prefer trimming text prior to matching against the ignore re. I know this is done wrong with other ones, but no reason to not do it more right.